src/pages/checkout/index.tsx

```diff
@@ -84,10 +84,9 @@ const Checkout: React.FC = () => {
 
 		api
```

Comment:
Не стесняйтесь пользоваться деструктуризацией) Она очень сильно может сократить код:

```suggestion
		api
			.usersOrderCreate(formData)
			.then(({ order_number }) => {
				navigate('/cart/success', { state: { order: order_number } });
				loadCartData();
			})
			.catch(({ response: { data } }) => {
				data?.errors
					? alert('Ошибка при создании заказа: ' + data.errors)
					: alert('Произошла ошибка при создании заказа.');
			});
```

Reply:
Окей)